CHANGELOG.md

```diff
@@ -6,6 +6,48 @@ are available [here](docs/development/system-tests.md#performance-benchmarks). A
 available [here](docs/development/roadmap.md).
 
 
+## Version 0.32.0
+
+*Note: this release contains breaking changes. It is not possible to upgrade from a previous version of Sleeper
+to version 0.32.0*
+
+This contains fixes to compaction with DataFusion, and preparation for deployment from published artefacts.
+
+Ingest:
+- All ingest systems (standard, bulk import, batcher) now allow filenames including a scheme such as s3a:// or s3://
+
+Compaction:
+- Experimental DataFusion aggregation support is now applied automatically for Java compactions
```

Comment:
This needs rewriting - if it's support for aggregations using DataFusion, how can they be applied in Java compactions? I think what it means is that the aggregation property is still applied even if the user chooses Java compactions.